docs/release_notes/pending_release.rst

```diff
@@ -1,29 +1,10 @@
-Pending Release Notes
-=====================
+SMQTK Pending Release Notes
```

Comment:
I've been not putting the `SMQTK` prefix in this since I think its super redundant when observed via the "SMQTK-*" titled docs or repo.